src/app/qgisapp.cpp

```diff
@@ -978,6 +1026,16 @@ QgisApp::QgisApp( QSplashScreen *splash, bool restorePlugins, bool skipVersionCh
 #ifdef ANDROID
   toggleFullScreen();
 #endif
+  QgsDebugMsg("PROFILE TIMES");
```

Comment:
Good idea.  I was also going to just have a few other "reporting" methods, e.g `max`, `top10hot`, `asHtml`, etc.

Maybe it also makes sense to store the data for the runs in a sqlite db with a data time stamp for the profile instance.